config/api.yml

```diff
@@ -540,6 +540,19 @@
         :identifier: instance_guest_restart
       - :name: reset
         :identifier: instance_reset
+  :notifications:
+    :description: User's past notifications
+    :options:
+    - :collection
+    - :arbitrary_resource_path
```

Comment:
@isimluk I don't think that should be blocking you, unless this collection has or is a subcollection(s)?